src/frontend/screens/Library/components/InstallModal/ModDialog/constants.ts

```diff
@@ -0,0 +1,17 @@
+export interface DownloadLinks {
+  chromieCraft: string
+  archiveOrg: string
+  webtorrent: string
+  magnet: string
+}
+
+export const downloadLinks: Record<string, DownloadLinks> = {
+  '0x70e6d87cd62a0a643e48cab789c47890248fe1e8d320f41ba8e1c31d90f97d81': {
+    chromieCraft: 'https://chromiecraft.com/download',
+    archiveOrg:
```

Comment:
this one is not downloading the correct files

Reply:
yeah, its just a placeholder until we figure it out the correct links.
I will comment the code for now